src/lib/style/components/_query-wrapper-plot-nav.scss

```diff
@@ -73,18 +41,58 @@
           grid-column: 1 / span 2;
         }
       }
-      .isHidingFacetFilters {
-        &.QueryFilter {
-          grid-column: 1 / span 1;
-          > * {
-            display: none;
-          }
-        }
-        &.FilterAndView {
-          margin-top: 10px;
-        }
+    }
+  }
+
+  // row
+  .SRC-wrapper {
+    .TopLevelControls {
+      grid-row: 1 / span 1;
+    }
+    .TotalQueryResults {
+      grid-row: 2 / span 1;
+    }
+    .download-confirmation {
+      grid-row: 3 / span 1;
+    }
+    .QueryWrapperSearchInput {
+      grid-row: 4 / span 1;
+      .MuiCollapse-entered {
+        margin: 5px 0px;
       }
     }
+    .QueryWrapperSqlEditorInput {
+      grid-row: 5 / span 1;
+      .MuiCollapse-entered {
+        margin: 5px 0px;
+      }
+    }
+    .ErrorBannerWrapper {
+      grid-row: 6 / span 1;
+    }
+    // This is the placement for mobile.  See above for desktop position
+    // (when showing these facet filters we render a 2 column layout)
+    .QueryFilter {
+      grid-row: 7 / span 1;
+    }
+    .FacetNav {
+      grid-row: 8 / span 1;
+    }
+    .FilterAndView {
+      grid-row: 9 / span 1;
+    }
+  }
+
+  .SRC-wrapper .isHidingFacetFilters {
```

Comment:
Moving this out is the main fix for PORTALS-2307 (apply display:none to QueryFilter children, when isHidingFacetFilters)